bin/branding.list

```diff
@@ -10,6 +10,7 @@ AUTHOR
 LICENSE
 HOMEPAGE
 CHANGELOG_PAGE
+ORGANIZATION
```

Comment:
While it's true that it is required for publishing... it's not actually used by @@VARIABLE@@ in the branding, it's used as a normal environment variable.  Safety checks in branding.py won't help us at all here. 

The pre-existing RPM_URL is used here in the build, for example: https://github.com/jenkinsci/packaging/blob/a610a9d26d588e7d717ff50d952d692eecfe972e/rpm/build/SOURCES/jenkins.repo

We're building out an anti-pattern here I think, long term.  If the publish fails with appropriate errors when given an invalid organization, that is probably sufficient for our needs.